deeplay/components/dcgan/dcgan.py

```diff
@@ -0,0 +1,464 @@
+from typing import List, Optional, Literal, Any, Sequence, Type, overload, Union
+
+from ... import (
+    DeeplayModule,
+    Layer,
+    LayerList,
+    Sequential,
+    LayerActivation,
+    LayerActivationNormalization,
+)
+
+import torch
+import torch.nn as nn
+
+from deeplay import LayerList
+
+
+class ConvBlock(DeeplayModule):
+    """
+    Convolutional block for DCGAN discriminator. It consists of a 2D Convolutional layer, a Batch Normalization layer and a LeakyReLU activation function.
+    """
+
+    def __init__(
+        self,
+        in_channels,
+        out_channels,
+        kernel_size,
+        stride,
+        padding,
+        batch_norm=True,
```

Comment:
argument should be `normalization` which accepts a deeplaymodule. Should also add argument for activation

Reply:
Corrected.